### PR TITLE
Migrate to use sbt-davenverse

### PR DESCRIPTION
**Pull request by JesusMtnez**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,39 +1,29 @@
 import sbtcrossproject.CrossPlugin.autoImport.crossProject
-import sbtghactions.UseRef
 
 val Scala213 = "2.13.6"
 val Scala212 = "2.12.14"
 val Scala3 = "3.0.0"
 
-Global / onChangedBuildSource := ReloadOnSourceChanges
```
Comment (ChristopherDavenport) [inline build.sbt:8]: Did you intend to lose this?

File: build.sbt [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
-    name = Some("Setup Ruby"),
-    params = Map("ruby-version" -> "2.6.0"),
-    cond = cond
+ThisBuild / startYear := Some(2018)
+ThisBuild / developers := List(
+  Developer(
+    "christopherdavenport",
+    "Christopher Davenport",
+    "chris@christopherdavenport.tech",
+    new java.net.URL("https://christopherdavenport.github.io/")
```
Comment (ChristopherDavenport) [inline build.sbt:17]: Pretty sure there is a `url"xyz"` interpolator for this

**Review by ChristopherDavenport — APPROVED** [review]
What do you think of the plugin? Mostly do what you expect/want?

**Comment by JesusMtnez** [comment]
> What do you think of the plugin? Mostly do what you expect/want?

I think it is doing a great job to unify all the settings and procedures we usually need to setup a new open source project, specially in `davenverse` ecosystem. I am not a huge fun of having a share dependency between all the projects, but in this case, if we keep it as small sbt plugins to setup things like release, github-actions, scalafmt, mima, ... so you could enable / disable as you need in your project, it would really help to maintain and reduce the copy / paste between projects.

**Pull request merged**